devel/libgit2/Portfile

```diff
@@ -11,7 +11,7 @@ conflicts           libgit2-devel
 set my_name         libgit2
 
 # don't forget to update py-pygit2 and libgit2-glib as well
-github.setup        libgit2 libgit2 1.6.4 v
+github.setup        libgit2 libgit2 1.7.0 v
 github.tarball_from archive
 revision            1
```

Comment:
I failed to set the revision back to zero, but will take care of before merging.

Reply:
Revision fixed